docs/getting_started.md

```diff
@@ -24,14 +26,63 @@ async def produce():
         await asyncio.sleep(5)
 
 
-async def main():
+async def start():
     await stream_engine.start()
     await produce()
+
+
+async def shutdown():
     await stream_engine.stop()
 
 
 if __name__ == "__main__":
-    asyncio.run(main())
+    loop = asyncio.get_event_loop()
+    try:
+        loop.run_until_complete(start())
+        loop.run_forever()
+    finally:
+        loop.run_until_complete(shutdown())
+        loop.close()
+```
+
+*(This script is complete, it should run "as is")*
+
+In the previous example you can see some boiler plate regarding how to start the program. We recommend to use [aiorun](https://github.com/cjrh/aiorun),
+so you want have to worry about `set signal handlers`, `shutdown callbacks`, `graceful shutdown` and `close the event loop`.
```

Comment:
Aiorun is good to run it in isolation, but uvicorn can also be used if fastapi is run in parallel, right?

Reply:
`uvicorn` is the ASGI web server implementation that will run the application, in this case`FastAPI/Starlette`. We only need aiorun when an `ASGI` application is **not** present.

So,  if `uvicorn` is running, the engine will use the event loop created by `uvicorn`  but not `uvicorn` itself.

Just in case, for `FastAPI` we have the usage: https://kpn.github.io/kstreams/getting_started/#fastapi.